js-closures.md

```diff
@@ -10,9 +12,9 @@ function makeFunc() {
 }
 
 var myFunc = makeFunc();
-myFunc();
-```
-
-See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures)
-
+myFunc();
+```
+
+See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures)
+
 > tags: closure, javascript, js
```

Comment:
I don't see any difference ere, he just added a heading, rich different shows just that. Just 2 new lines.